terms.html

```diff
@@ -5,12 +5,18 @@
 <dl class="termlist">
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
+A digital representation of an assertion made about an <a>entity</a> by a specific <a>entity</a>.
+The entity about which the assertion is made is called the <a>subject</a> of the claim.
+The meaning of the claim (its semantics) is determined by its author.
+Claims can be authored by <a>issuers</a>, e.g., to construct a <a>verifiable credential</a> with, or 
+by <a>holders</a>, e.g., to include in a <a>verifiable presentation</a>.
+Different authors can reuse the same semantics for claims through the use of shared vocabularies.
   </dd>
   <dt><dfn data-lt="credential|credentials">credential</dfn></dt>
   <dd>
-A set of one or more <a>claims</a> made by an <a>issuer</a>. 
-The <a>claims</a> in a credential can be about different <a>subjects</a>.
+A set of one or more <a>claims</a> created by a single <a>entity</a>,
+that is constructed for the purpose of transferring it to some other <a>entity</a>. 
```

Comment:
```suggestion
A set of one or more <a>claims</a> created by a single <a>issuer</a>,
that is constructed for the purpose of transferring it to a <a>holder</a>. 
```

Reply:
@brentzundel: agree with the change to 'issuer'. 

@brentzundel and @TallTed: Regarding the change to 'holder', 
1. the current definition of 'holder' is about entities posessing VCs. The change would (strictly speaking) imply that in a situation where entities don't yet  possess VCs, they are not holders and issuers could then not intend to transfer VCs to them. 
2. The point here that VCs are intended to be transferred. Whereto, under what circumstances etc. seems secondary to me. Keeping 'entity' seems therefore more appropriate to me.

Reply:
`entity` appears nowhere in our `issuer`-`holder`-`verifier` model.

A `holder` becomes such upon receipt of a VC, if they weren't already. I see no problem with this.